lib/code_corps_web/controllers/github_event_controller.ex

```diff
@@ -50,9 +50,9 @@ defmodule CodeCorpsWeb.GithubEventController do
     action = payload |> Map.get("action", "")
     event_support =
       if should_process?(payload) do
-        type
-        |> EventSupport.status(action)
-        |> process_event(type, delivery_id, payload)
+        process_status = type |> EventSupport.status(action)
+        process_status |> process_event(type, delivery_id, payload)
+        process_status
       else
         :ignored
       end
```

Comment:
So will this line right now never run, the `:ignored` line?

Reply:
Nevermind, I get it with the above comment. Should've waited.

Reply:
The `:ignored` line will be executed in the case where the webhook event includes `%{"repository" => %{"id" => _}}` but the associated `GithubRepo` does not contain an associated `project_id` in the DB.